README.md

```diff
@@ -1,15 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Awesome Books
 
-> Description the project.
+> A online application that allows you to add your books to a digital shelf. Vanila javascript was used to create this website. This project now has a local storage option, allowing users to save data in the web app.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
 
 ## Live Demo (if available)
 
```

Comment:
- [ ] OPTIONAL: You can delete `if available` note. Because it simply means that if the live demo is not available you should not include this section

Reply:
Thank you for your comment. I like your GIF by the way.😂 Really funny.

Reply:
@ShoiraTa Tank you for your approval👍🏼